tests/fixtures/webext-as-library/test-import.mjs

```diff
@@ -0,0 +1,7 @@
+// eslint-disable-next-line import/no-unresolved
+import webExt from 'web-ext';
+
+// eslint-disable-next-line import/extensions
+import _ from './helpers.js';
```

Comment:
> @rpl I have a doubt about imports:
> When I tried importing helper.js the following way:
>
> import _ from './helpers';
>
> It gave me an error and the test of importing web-ext as ESM module failed
> But when I used:
>
> import _ from './helpers.js';
>
> the test passes.
> Why is that so?
> Edit: I did some reading on it. So to my understanding first import statement fails because it looks for helpers directory instead of the file. Please correct me if I am wrong.

The reason behind that is likely the nodejs conventions related to ESM modules, in short nodejs looks by default for files with extension `.mjs` when we use `import` instead of `require` (I think you may find some more details about in the original announcement here: https://nodejs.medium.com/announcing-a-new-experimental-modules-1be8d2d6c2ff)

and so:
`import ... from './helper'`
I guess that would be looking for `./helpers.mjs` and `./helpers/index.mjs`, and it fails if neither of them is found.

on the contrary `import ... from './helpers.js'` is explicitly specifying the extension and will override that convention.